doc/Makefile

```diff
@@ -28,20 +26,20 @@ api:
 	@echo
 	$(SPHINXAUTOGEN) -i -t _templates -o api/generated api/*.rst
 
-html: api
+html latex latexpdf: api
```

Comment:
`latex` and `latexpdf` are added here to support PDF documentation (xref #1606).